configs/components/echam/echam.datasets.yaml

```diff
@@ -11,6 +11,12 @@ input_sources:
         "vltclim": "${adj_input_dir}/${resolution}${ocean_resolution}_VLTCLIM.nc"
 
 # forcings
+# Note: The variable ``ozone_source_file_scenario`` is set to ${scenario} by default.
+# In case you are running a HIST scenario, change the value of this variable to 
+# ssp126, ssp119, ssp245, ssp370, or ssp585
```

Comment:
```suggestion
```
I think is best to, instead the users doing it manually, for it to happen automatically: set `ozone_source_file_scenario: ssp370` in the echam.yaml inside the HIST scenario choose block. ssp370 is what I heard people is using for the last year in Historical simulations for 2015.